### ISSUE TITLE
NoSuchMethodError

### ISSUE THREAD

**Issue by SlimeDog** [MostWantedMinx/My-Plugins#5]
Spigot 1.13.1 latest
PRP 1.1.0
WorldGuard 7.0.0-b1744

AllStormsAcid:true is configured in this world. I executed
> /storm

and acid rain commenced. Console/log spammed with:
```
[15:40:23 WARN]: [PyroWeatherPro] Task #6963 for PyroWeatherPro v1.1.0 generated an exception
java.lang.NoSuchMethodError: com.sk89q.worldguard.bukkit.WorldGuardPlugin.getRegionManager(Lorg/bukkit/World;)Lcom/sk89q/worldguard/protection/managers/RegionManager;
        at me.arsmagica.WeatherTypes.AcidRain$5.run(AcidRain.java:273) ~[?:?]
        at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:76) ~[spigot-1.13.1.jar-2018-09-04-1018:git-Spigot-0ede7d0-c88ae6b]
        at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:361) [spigot-1.13.1.jar-2018-09-04-1018:git-Spigot-0ede7d0-c88ae6b]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:889) [spigot-1.13.1.jar-2018-09-04-1018:git-Spigot-0ede7d0-c88ae6b]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:411) [spigot-1.13.1.jar-2018-09-04-1018:git-Spigot-0ede7d0-c88ae6b]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:831) [spigot-1.13.1.jar-2018-09-04-1018:git-Spigot-0ede7d0-c88ae6b]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:729) [spigot-1.13.1.jar-2018-09-04-1018:git-Spigot-0ede7d0-c88ae6b]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
```

**Comment by SlimeDog** [comment]
From #7: 
These were generated every second when I connected (in survival mode). They stopped when I changed to creative mode. I know, just warnings, but still... filling the console/log.

**Comment by MostWantedMinx** [comment]
I might need to update the API in the plugin. Just found out that I'm using quite an old version of the world guard API. I'll update this in my plugin folder and see if that fixes the error. It does however work on 1.12.2.

**Comment by SlimeDog** [comment]
APIs have changed. Agree that it worked on 1.12.2.

**Comment by SlimeDog** [comment]
I can only report for 1.13.1. I am not testing 1.13. Bleeding edge WG version is listed above: WorldGuard 7.0.0-b1744

**Comment by MostWantedMinx** [comment]
Not sure what I am doing wrong but when I'm trying to load that world guard version, it gives an error in console and does not let me use the plugin. What World Edit and or FAWE version are you using?

**Comment by SlimeDog** [comment]
WorldEdit version 7.0.0-b3861

**Comment by MostWantedMinx** [comment]
Interesting, even with these two exact versions I'm receiving errors in my console and they do not enable. Unsure how to test this without having the actual plugins being enabled.

**Comment by SlimeDog** [comment]
I have no good idea how to proceed. Here is my complete plugin list, for reference. All are bleeding edge; this is a test server. Please let me know if anything stands out.
```
[12:41:52 INFO]: AdminVault               0.9
[12:41:52 INFO]: AntiAFK                  1.1
[12:41:52 INFO]: Arconix                  2.0.0
[12:41:52 INFO]: BentoBox                 0.9.0-alpha2
[12:41:52 INFO]: BiomeFinder              1.2.9
[12:41:52 INFO]: BloodManager             1.7
[12:41:52 INFO]: BookExploitFix           0.11
[12:41:52 INFO]: Chain                    1.0.3
[12:41:52 INFO]: CHIP                     0.4.9
[12:41:52 INFO]: Citizens                 2.0.24-SNAPSHOT (build 1568)
[12:41:52 INFO]: CombatLogX               9.0.0.0
[12:41:52 INFO]: CommandDoesNotExist      2.8.0
[12:41:52 INFO]: CommandSigns             1.6.7
[12:41:52 INFO]: CustomShields            1.4.5
[12:41:52 INFO]: DelayJoin                1.0-SNAPSHOT
[12:41:52 INFO]: DeluxeMenus              1.10.5
[12:41:52 INFO]: Denizen                  1.0.3-SNAPSHOT (build 411)
[12:41:52 INFO]: Depenizen                1.0.0 (build 415)
[12:41:52 INFO]: DisableItemFrameRotation 1.2.0
[12:41:52 INFO]: dynmap                   3.0-beta-1-129
[12:41:52 INFO]: Dynmap-Citizens2         0.40
[12:41:52 INFO]: Dynmap-Essentials        0.91-SNAPSHOT
[12:41:52 INFO]: Dynmap-GriefPrevention   0.80
[12:41:52 INFO]: dynmap-structures        1.5.1-SNAPSHOT
[12:41:52 INFO]: Dynmap-WorldGuard        1.1-alpha-1
[12:41:52 INFO]: Elevators                1.4.1
[12:41:52 INFO]: EntityList               1.0
[12:41:52 INFO]: Essentials               2.15.0.45
[12:41:52 INFO]: EssentialsChat           2.15.0.45
[12:41:52 INFO]: ExtraContexts            1.0-SNAPSHOT
[12:41:52 INFO]: GriefPrevention          16.9
[12:41:52 INFO]: HeadsInventory           1.7.1-1.13
[12:41:52 INFO]: HeadsPluginAPI           1.7.1-1.13
[12:41:52 INFO]: HeadSweeper              1.4.0-1.13
[12:41:52 INFO]: HolographicDisplays      2.3.1-SNAPSHOT
[12:41:52 INFO]: HungerKeeperPlus         1.4.1
[12:41:52 INFO]: ItemJoin                 4.9.1-SNAPSHOT
[12:41:52 INFO]: LightAPI                 3.2.7
[12:41:52 INFO]: LuckPerms                4.2.99
[12:41:52 INFO]: MazeCraft                1.0.2
[12:41:52 INFO]: MessageHacker            1.3.0
[12:41:52 INFO]: MobHunting               6.0.6
[12:41:52 INFO]: Multiverse-Core          2.6.0-b738
[12:41:52 INFO]: Multiverse-Inventories   2.5.0-SNAPSHOT-b444
[12:41:52 INFO]: Multiverse-NetherPortals 2.5.2-SNAPSHOT-b729
[12:41:52 INFO]: Multiverse-Portals       2.5.2-SNAPSHOT-b765
[12:41:52 INFO]: NetworkInterceptor       1.0-SNAPSHOT
[12:41:52 INFO]: PandaWire                1.0.8-SNAPSHOT
[12:41:52 INFO]: PhantomX                 1.2
[12:41:52 INFO]: PhatLoots                4.2.3+79
[12:41:52 INFO]: Pl3xMOTD                 3.0
[12:41:52 INFO]: PlaceholderAPI           2.9.2
[12:41:52 INFO]: Plan                     4.4.5-b1
[12:41:52 INFO]: PluginVersions           1.0.0
[12:41:52 INFO]: ProtocolLib              4.4.0-SNAPSHOT-b415
[12:41:52 INFO]: PyroWeatherPro           1.1.2
[12:41:52 INFO]: RedstoneClockDetector    1.2.1
[12:41:52 INFO]: RedstoneProximitySensor  2.3.9
[12:41:52 INFO]: Sentinel                 1.6.2 (build 189)
[12:41:52 INFO]: ServerListPlus           3.4.8
[12:41:52 INFO]: ServerTutorialPlus       1.19.0
[12:41:52 INFO]: ShinyItems               1.6.3
[12:41:52 INFO]: Shopkeepers              2.4.1
[12:41:52 INFO]: SignBoard                1.5.0
[12:41:52 INFO]: SignEditor               1.07
[12:41:52 INFO]: Skript                   2.3-alpha3
[12:41:52 INFO]: spark                    1.0.3
[12:41:52 INFO]: TabList                  3.3
[12:41:52 INFO]: Telecom                  0.1.1
[12:41:52 INFO]: Thirst                   2.8.5
[12:41:52 INFO]: TreeAssist               5.13.1
[12:41:52 INFO]: UltimateFirework         1.4.6
[12:41:52 INFO]: Vault                    1.7.1-b91
[12:41:52 INFO]: VoidGenerator            1.3.1
[12:41:52 INFO]: VoidSpawn                1.17.0
[12:41:52 INFO]: WDLCompanion             1.2.0
[12:41:52 INFO]: WhatPlugin               1.0-SNAPSHOT (build #3)
[12:41:52 INFO]: WorldEdit                7.0.0-SNAPSHOT;3861-6a71cd2
[12:41:52 INFO]: WorldGuard               7.0.0-SNAPSHOT;1744-8e819f7
[12:41:52 INFO]: WorldResourcepacks       1.6.19-SNAPSHOT (build 224)
```
and WG regions.yml for this world
```
regions:
    spawn:
        min: {x: 131.0, y: 50.0, z: 230.0}
        max: {x: 141.0, y: 125.0, z: 241.0}
        members: {}
        flags:
            creeper-explosion: deny
            enderdragon-block-damage: deny
            enderman-grief: deny
            entity-item-frame-destroy: deny
            entity-painting-destroy: deny
            fire-spread: deny
            ghast-fireball: deny
            mob-damage: deny
            mob-spawning: deny
            other-explosion: deny
        owners:
            unique-ids: [3cf9b867-2577-49d1-affb-7ba8196a0d39]
        type: cuboid
        priority: 100
    __global__:
        members: {}
        flags: {}
        owners: {}
        type: global
        priority: 0
```
Seems pretty tame. And for this test I was not in the spawn area, anyway.

**Comment by MostWantedMinx** [comment]
Your plugin list looks completely fine to me. I'll try have another crack at setting it up later tonight.

**Comment by MostWantedMinx** [comment]
I think I have finally got it working on my test server. Will update this post when I figure out if I can recreate the bug.

**Comment by SlimeDog** [comment]
As stated in the discussion thread:

_Suggestions:_
- _Leave available for download the last version that works on 1.12.2, so those who want to run 1.12.2 have it_
- _Move forward with 1.13.x_

Other plugins seem to support multiple MC versions with reflection. Others leave older plugin versions available for older MC versions. It's a developer choice.

**Comment by MostWantedMinx** [comment]
I'll have a think about this over the next few hours. I think what I am most likely to do, is to release

**Comment by MostWantedMinx** [comment]
- Fixed in PyroWeatherPro (1.13.X) v1.1.3

**Comment by SlimeDog** [comment]
Confirmed fix in PWP 1.1.3 (1.13.x version).
